core/src/test/java/org/kohsuke/stapler/DataBindingTest.java

```diff
@@ -79,6 +79,7 @@ public String getContentType() {
         RequestImpl req = new RequestImpl(new Stapler(), mr, Collections.<AncestorImpl>emptyList(), null);
         new Function.InstanceFunction(getClass().getMethod("doFromStaplerMethod",StaplerRequest.class,int.class,Binder.class))
                 .bindAndInvoke(this,req,null);
+        assertEquals(42, new Function.InstanceFunction(getClass().getMethod("doStaticMethod")).bindAndInvoke(this, req, null));
```

Comment:
Yes it is weird to be calling an `InstanceFunction` here, but apparently that is what form validation does. (Trying to use `StaticFunction` here does not work—you get an `ArrayIndexOutOfBoundsException`. Unclear if this is related to unreviewed change by @kohsuke in bd67c1eda4d096c0943a37ff1e84f028bf6348f5.)